cukinia/common_security_tests.d/hardening.conf

```diff
@@ -55,3 +55,9 @@ as "SEAPATH-00100 - /etc/securetty is empty" not cukinia_test -s /etc/securetty
 as "SEAPATH-00101 - PAM securetty module is active in 'login' policy" \
     cukinia_cmd grep -qE "^[^#].*pam_securetty.so" /etc/pam.d/login
 
+# Check that bootloader is password protected with a superuser "root" & one
+# unrestricted boot menuentry
+as "BP-028-R5-1 - grub root superuser is set in /boot/grub/grub.cfg" \
+	cukinia_cmd grep -q "superusers=root" /boot/grub/grub.cfg
```

Comment:
this could be inside a comment. Check that it's not a comment with `^superusers=root`